tests/Tests/600__ai_apps/640__rhoam/640__rhoam_installation.robot

```diff
@@ -0,0 +1,84 @@
+*** Settings ***
+Resource        ../../../Resources/Page/LoginPage.robot
+Resource        ../../../Resources/Common.robot
+Resource        ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource        ../../../Resources/Page/ODH/AiApps/Rhoam.resource
+Resource        ../../../../tasks/Resources/RHODS_OLM/uninstall/uninstall.robot
+Resource        ../../../../tasks/Resources/RHODS_OLM/uninstall/oc_uninstall.robot
+Resource        ../../../../tasks/Resources/RHODS_OLM/config/cluster.robot
+Library         ../../../../libs/Helpers.py
+Library         SeleniumLibrary
+Library         OpenShiftLibrary
+Suite Setup     RHOAM Suite Setup
+Suite Teardown  RHOAM Suite Teardown
+
+
+*** Test Cases ***
+Verify RHOAM Can Be Installed
+    [Documentation]    Verifies RHOAM Addon can be successfully installed
+    [Tags]  Tier3
+    ...     ODS-273
+    ${cluster_id}=   Get Cluster ID
+    ${cluster_name}=   Get Cluster Name     cluster_identifier=${cluster_id}
+    Install Rhoam Addon    cluster_name=${cluster_name}
+    Wait Until RHOAM Installation Is Completed    retries=35   retries_interval=2min
+    Verify RHOAM Is Enabled IN RHODS Dashboard
+
+Verify RHODS Can Be Uninstalled When RHOAM Is Installed
+    [Documentation]    Verifies RHODS can be successfully uninstalled when
+    ...                RHOAM addon is installed on the same cluster
+    [Tags]  Tier3
+    ...     ODS-1136
+    ${cluster_id}=   Get Cluster ID
+    ${CLUSTER_NAME}=   Get Cluster Name By ID     cluster_id=${cluster_id}
+    Set Suite Variable     ${CLUSTER_NAME}
+    Verify RHOAM Is Enabled In RHODS Dashboard
+    Uninstall RHODS From OSD Cluster
+    Wait Until RHODS Uninstallation Is Completed
+
+
+*** Keywords ***
+RHOAM Suite Setup
+    [Documentation]    RHOAM Suite setup
+    Set Library Search Order  OpenShiftLibrary  SeleniumLibrary
+    # Set Library Search Order  OpenShiftLibrary
+
+RHOAM Suite Teardown
+    [Documentation]    RHOAM Suite teardown. It triggers RHOAM Uninstallation
+    Log To Console    Starting uninstallation of RHOAM Addon...
+    Uninstall Rhoam Addon    cluster_name=${CLUSTER_NAME}
+    Log To Console    RHOAM Addon has been uninstalled!
+    Close All Browsers
+
+Uninstall RHODS From OSD Cluster
```

Comment:
This keyword could be handy for other tests. Maybe it could go in tests/Resources/ODS.robot